### PR TITLE
update irc references to point to libera.chat

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: puppetdb_foreman.gemspec [hunk 1]
```diff
@@ -7,7 +7,7 @@ Gem::Specification.new do |s|
   s.version     = PuppetdbForeman::VERSION
   s.license     = 'GPL-3.0'
   s.summary     = 'This is a Foreman plugin to interact with PuppetDB.'
-  s.description = 'Disable hosts on PuppetDB after they are deleted or built in Foreman. Follow https://github.com/theforeman/puppetdb_foreman and raise an issue/submit a pull request if you need extra functionality. You can also find some help in #theforeman IRC channel on Freenode.'
+  s.description = 'Disable hosts on PuppetDB after they are deleted or built in Foreman. Follow https://github.com/theforeman/puppetdb_foreman and raise an issue/submit a pull request if you need extra functionality. You can also find some help in #theforeman IRC channel on Libera.Chat.'
```
Comment (ekohl) [inline puppetdb_foreman.gemspec:10]: Made me think of https://github.com/theforeman/puppet-dns/pull/222#issuecomment-1268390797. Should we point them to https://theforeman.org/support.html?

**Review by timogoebel — APPROVED** [review]

File: puppetdb_foreman.gemspec (hunk 1 above)
Comment (evgeni) [reply puppetdb_foreman.gemspec:10]: Did you just use my argument against me?

But yes, now that you said it, it's probably a good idea.

@timogoebel whatcha think? (also: hi! :wave: )

Comment (timogoebel) [reply puppetdb_foreman.gemspec:10]: Link to the support page sounds like the best solution to me as it's probably future proof.

@evgeni: Hi, yourself! And... don't tell anyone you saw me here. ;-)

**Review by ekohl — APPROVED** [review]
Thanks!

File: puppetdb_foreman.gemspec [hunk 2]
```diff
@@ -7,7 +7,7 @@ Gem::Specification.new do |s|
   s.version     = PuppetdbForeman::VERSION
   s.license     = 'GPL-3.0'
   s.summary     = 'This is a Foreman plugin to interact with PuppetDB.'
-  s.description = 'Disable hosts on PuppetDB after they are deleted or built in Foreman. Follow https://github.com/theforeman/puppetdb_foreman and raise an issue/submit a pull request if you need extra functionality. You can also find some help in #theforeman IRC channel on Freenode.'
+  s.description = 'Disable hosts on PuppetDB after they are deleted or built in Foreman. Follow https://github.com/theforeman/puppetdb_foreman and raise an issue/submit a pull request if you need extra functionality. You can also find some help via the Foreman support pages (https://theforeman.org/support.html).
```
Comment (ekohl) [inline puppetdb_foreman.gemspec:10]: Of course I spot something the moment I press submit.
```suggestion
  s.description = 'Disable hosts on PuppetDB after they are deleted or built in Foreman. Follow https://github.com/theforeman/puppetdb_foreman and raise an issue/submit a pull request if you need extra functionality. You can also find some help via the Foreman support pages (https://theforeman.org/support.html).'
```

**Comment by timogoebel** [comment]
Danke!

**Pull request merged**